SocialFilter/sections/allowlist.txt

```diff
@@ -16,6 +16,8 @@ designervn.net#@#.fb-share-button
 !#endif
 ! #End Social locker
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/170566
+105.net#@#.social2
```

Comment:
better remove blocking rule and another exclusion - it does not look good.

Reply:
Ok, done 😊